docs/local-tableland/fork-chain.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Forking chain state
+description: Fork a live testnet or mainnet chain and use its state on Local Tableland.
+keywords:
+  - fork
+  - fork chain
+  - local tableland
+---
+
+A common development pattern is to use an existing chain's state as a starting point for testing. Local Tableland supports forking a live testnet or mainnet chain, and the full chain's state is available for use. This includes any smart contract or transactions that occurred on the chain, and the Tableland table state materialized is also materialized on Local Tableland. In other words, you can interact with some table deployed on the forked, live chain as if it were a local table.
+
+## Overview
+
+The required parameters to fork a chain are passed during `LocalTableland` instantiation:
+
+- `forkUrl`: The provider URL of the chain to fork, such as an Alchemy or Infura URL.
+- `forkBlockNumber`: The block number to fork from, which recreates the chain state starting at that block.
+- `forkChainId`: The chain ID of the chain to fork.
+
+Once these values are set, a few things will happen:
+
+- The chain's forked state will be initialized at the starting block number.
+- Events/logs will be replayed from the forked chain, starting from the block number specified.
+- The Tableland validator will process _all_ of the Tableland registry's events/logs from the forked chain up until the forked block, and the state of the tables will be materialized.
+
+When you initially start Local Tableland, it will take a bit of time to backfill the forked chain's state. The `forkUrl` you specify for the provider will make API calls to `eth_getLogs` in batches, and the validator replays these one-by-one. Keep in mind this will continue hitting the API until the state is fully backfilled, but it _should_ be cached after the first time you run it.
```

Comment:
Might be good to include that caching only works if you use the same `forkBlockNumber` value each time you start LT?